.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "presets": ["es2015-webpack", "react"],
+  "presets": ["es2015-native-modules", "react"],
```

Comment:
This should be `["es2015", {"modules": false}]`. See [deprecation notice](https://github.com/araphel/babel-preset-es2015-native-modules#deprecated-notice)